autoload/merginal/buffers/base.vim

```diff
@@ -1,6 +1,7 @@
 call merginal#modulelib#makeModule(s:, 'base', '')
 
 let s:f.helpVisible = 0
+let s:f.isRemoteVisible = 0
```

Comment:
1. For consistency, this should be just `s:f.remoteVisible`.
2. To avoid changing the default behavior, this should be configurable and default to showing the remote branches:
    ```vim
    let s:f.remoteVisible = get(g:, 'merginal_remoteVisible', 1)
    ```
    If you do add this setting, please document it in `doc/merginal.txt`.